code-scanning/scorecards.yml

```
@@ -0,0 +1,73 @@
+name: Scorecards supply-chain security
+on: 
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    # Weekly on Saturdays.
+    - cron: '30 1 * * 6'
```

We generally use this template variable in order to randomise the schedule, and distribute workflow load more across the week.

Users are of course free to edit this to whatever they choose, but is there a reason it needs to have this specific default?
```suggestion
    - cron: $cron-weekly
```

---

I'll change this back, thanks!

---

Done.